site/public/js/manage-graders-columns.js

```diff
@@ -0,0 +1,68 @@
+/* exported toggleColumnsForm, updateManageGradesColumns, fillAllCheckboxes
+*/
+
+//Data structure for active columns
+const checkboxes = document.getElementsByClassName('toggle-columns-box');
+
+//opens modal with initial settings for new student
+function toggleColumnsForm() {
+    const form = $('#toggle-columns-form');
+    form.css('display', 'block');
+    checkProperTicks();
+}
+
+//checks proper tick marks in modal
+function checkProperTicks() {
+    const selectedColumns = loadColumns();
+    for (let i = 0; i<checkboxes.length; i++) {
+        if (selectedColumns[i] === 1) {
+            checkboxes[i].checked = true;
+        }
+        else {
+            checkboxes[i].checked = false;
+        }
+    }
+}
+
+function updateManageGradesColumns() {
```

Comment:
very small nit-pick, but could this be changed to `updateManageGradersColumns()`?